.github/pull_request_template.md

```diff
@@ -1,14 +1,23 @@
-<!-- Please use a meaningful pull request title -->
-<!-- Please apply meaningful GitHub Labels to your pull request such as: PHP, Twig, SCSS, JS, etc. -->
+<!-- INSTRUCTIONS
+- Please use a meaningful pull request title which does not include the issue key or branch name.
+- Please apply meaningful GitHub Labels to your pull request such as: PHP, Twig, SCSS, JS, etc.
+- Please make sure you've reviewed the "Files changed" tab before opening this PR to ensure it includes what you expect (and nothing more) and adheres to our coding standards.
```

Comment:
I'm surprised _I'm_ the one saying this to _you_, but I think this long line should be wrapped @marcaddeo  😝 

```
- Please make sure you've reviewed the "Files changed" tab before opening this PR to ensure it
  includes what you expect (and nothing more) and adheres to our coding standards.
```

Reply:
Ha. Yeah it should. I didn't wrap it before because we had 1 comment per line.